kernel/driver.cc

```diff
@@ -674,6 +678,29 @@ int main(int argc, char **argv)
 			}
 			log("%s\n", out_count ? "" : " no commands executed");
 		}
+		if(!perffile.empty())
+		{
+			FILE *f = fopen(perffile.c_str(), "wt");
+			if (f == nullptr)
+				log_error("Can't open performance log file for writing: %s\n", strerror(errno));
+
+			fprintf(f, "{\n");
+			fprintf(f, "  \"generator\": \"%s\",\n", yosys_version_str);
+			fprintf(f, "  \"total_ns\": %lu,\n", total_ns);
```

Comment:
`#include <inttypes.h>` or whatever C++ wants, and then I think the portable solution is:
```suggestion
			fprintf(f, "  \"total_ns\": %" PRId64 ",\n", total_ns);
```